_includes/tutorials/finding-distinct/ksql/markup/dev/transient-query.adoc

```diff
@@ -0,0 +1,11 @@
+Note that we called the query that find only distinct events as `DETECTED_CLICKS` and we modeled as a table, since we are performing some aggregations in the query. Then we perform a rekeying to ensure that the `IP_ADDRESS` is used as key of the record, by transforming the table back to a stream and using the `PARTITION BY` clause. To verify if everything is working as expected, run the following query:
```

Comment:
```suggestion
In the first statement above, we created the query that finds only distinct events, naming it `DETECTED_CLICKS`. We modeled it as a table since the query performs aggregations.

Then, with the next two statements, we specified a re-keying to ensure that the `IP_ADDRESS` is used as key of the record, by transforming the table back to a stream and using the `PARTITION BY` clause.

To verify everything is working as expected, run the following query:
```